pkgs/applications/graphics/pizarra/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitLab
+, rustPlatform
+, cargo
+, pkg-config
+, binutils-unwrapped
+, gtk3-x11
+, atk
+, glib
+, librsvg
+, gdk-pixbuf
+, wrapGAppsHook
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "pizarra";
+  version = "v1.7.3";
+
+  src = fetchFromGitLab {
+    owner = "categulario";
+    repo = "pizarra-gtk";
+    rev = version;
+    sha256 = "sha256-XP+P2w6s47JQV4spKeMKe/Ktxid7uokGYH4IEJ5VHSc=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook glib pkg-config gdk-pixbuf ];
+
+  buildInputs = [ gtk3-x11 atk glib librsvg];
+
+  cargoSha256 = "sha256-fVOaEGdSnunhnoXRPV0JdXxSf5QaQGlcoKJhXuQiPzM=";
+
+  meta = {
+    description = "A simple blackboard written in GTK";
+    longDescription = ''
+      A simple endless blackboard.
+      Contains various features, such as:
+      - Pencil
+      - Rectangle
+      - Ellipse
+      - Line
+      - Text
+      - Grids
+    '';
+    homepage = "https://pizarra.categulario.tk/en/";
+    license = lib.licenses.gpl2Plus;
+    maintainers = with lib.maintainers; [ ];
```

Comment:
Please add yourself as a maintainer

Reply:
I have added myself to the `maintainers/maintainer-list.nix`.
I hope that's the recommended way.